src/main/memorygame/app.cljs

```diff
@@ -3,10 +3,16 @@
     [reagent.core :as r]
     [reagent.dom :as rdom]))
 
+(defn generer-billedrækken [x & cheat]
+      (if cheat
+        (interleave (range 1 (inc x)) (range 1 (inc x)))
+
+        (shuffle (concat (range 1 (inc x)) (range 1 (inc x))))))
+
 (def a (r/atom nil))
-(def billedrækken-2 (r/atom (zipmap (range 1 17) (shuffle (concat (range 1 9) (range 1 9))))))
-(def billedrækken (r/atom (zipmap (range 1 17) (concat (range 1 9) (range 1 9)))))
-(def tilstand (atom {
+
+(def billedrækken (r/atom (zipmap (range 1 17) (generer-billedrækken 8 :cheat ))))
+(def tilstand (r/atom {
                      :kort         nil
 
                      :næstespiller :player1
```

Comment:
looks good mate. :naestespiller needs to start on an even number for the game logic to work in the function spil. Can use this to show next player in the scoreboard: (if (even? (@tilstand :næstespiller)) "Player 1" "Player 2")